draft-ietf-quic-manageability.md

```diff
@@ -58,9 +58,10 @@ informative:
 --- abstract
 
 This document discusses manageability of the QUIC transport protocol, focusing
-on caveats impacting network operations involving QUIC traffic. Its intended
-audience is network operators, as well as content providers that rely on the use
-of QUIC-aware middleboxes, e.g. for load balancing.
+on the implications of QUIC's design and wire image on network operations
+involving QUIC traffic. Its intended audience is network and service
+operators who rely on the use of transport-aware network functions, e.g. for
+security or load balancing, as well as the implementors of those functions.
```

Comment:
```suggestion
operators who rely on the use of transport-aware network functions and the
implementors of those functions.
```

"e.g." in an abstract seems wrong to me.  I would say that using "for example" here is fraught because of the sheer diversity of transport-aware network functions.  Security, troubleshooting, capacity planning, load balancing, flow management, accounting, and more would be closer to a representative list, but I fear even that wouldn't be enough.

Reply:
Agreed, reluctantly.